Lib/http/server.py

```diff
@@ -150,7 +152,6 @@ def server_bind(self):
 class ThreadingHTTPServer(socketserver.ThreadingMixIn, HTTPServer):
     daemon_threads = True
 
```

Comment:
and let's back empty line (no need to make unnecessary style changes, this complicates the review)
```suggestion


```